core/storage/suggestion/gae_models.py

```diff
@@ -484,13 +484,43 @@ def get_in_review_translation_suggestions(
             list(SuggestionModel). A list of suggestions that are of the given
             type, which are in review, but not created by the given user.
         """
+
         return cls.get_all().filter(datastore_services.all_of(
             cls.status == STATUS_IN_REVIEW,
             cls.suggestion_type == feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
             cls.author_id != user_id,
             cls.language_code.IN(language_codes)
         )).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
 
+    @classmethod
+    def get_in_review_translation_suggestions_with_exp_ids(
+            cls, user_id: str, language_codes: List[str],
+            exp_ids: List[str]
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Gets all translation suggestions which are in review
+        and belong to the passed exp_ids.
+
+        Args:
+            user_id: str. The id of the user trying to make this query.
+                As a user cannot review their own suggestions, suggestions
+                authored by the user will be excluded.
+            language_codes: list(str). The list of language codes.
+            exp_ids: list(str). Exploration IDs matching the target ID of the
+                translation suggestions.
+
+        Returns:
+            list(SuggestionModel). A list of suggestions that are of the given
+            type, which are in review, not created by the given user and
```

Comment:
What do you mean by "given type"?

Should also mention dependence on the given language codes.

Reply:
Type here refers to whether it's suggestion type translate content or suggestion type add question. 

Sorry, missed this since language codes and type weren't part of the docstring earlier either, so I thought there might be a reason why it was skipped and didn't specify either.

Reply:
But this method is only for translations; no type is given.

Reply:
You are right. It looks the docstring is not right even for the pre-existing function (i.e. get_in_review_translation_suggestions)  since I re-used the return type explanation from there in order to avoid any confusion (Ref: https://github.com/oppia/oppia/blob/develop/core/storage/suggestion/gae_models.py#L530). 

I will fix that as well.

Reply:
Done and fixed pre-existing comment as well.